# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/pixman/hotfix-arm64-neon.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- /dev/null	2022-10-27 17:52:41.612390745 +0200
+++ pixman-0.42.0/a64-neon-test.S	2022-10-31 11:24:27.310483251 +0100
@@ -0,0 +1,2 @@
+
+
